readme.md

```diff
@@ -269,6 +269,21 @@ The following properties are defined:
 * `endpoint`: Base URL for the API, without e.g. the `index/graphql` suffix.
 * `apiVersion`: One of `otp1`, `otp2` or `entur`.
 
+#### TRIAS
+
+*WIP*
+
+The following properties are defined:
+* `endpoint`: Base URL for the API
+* `testingEndpoint`: Base URL for testing purposes
+* `triasVersion`: Highest [TRIAS version](https://github.com/VDVde/TRIAS/releases) supported by the API
+* `request-content-type`: [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) e.g. `text/xml`
```

Comment:
Now this kebab-case style is different from `requestLimitPolicy`'s camelCase style. I suggest to change one of them.